client/new-stack-editor/lib/index.coffee

```diff
@@ -25,12 +33,19 @@ module.exports = class StackEditorAppController extends AppController
 
     super options, data
 
+    @templates = {}
```

Comment:
Do you really need these to be saved locally? We already have a cache on `reactor` singleton, and all the other parts we have developed is using that as a source of truth. Loading stack templates here locally, will make other parts of the system unaware that this instance is loaded.